### PR TITLE
Multiple payload support

### PR DESCRIPTION
**Pull request by jsingle**
@paetling haven't tested that this works, but the idea look OK?

### CODE REVIEW

**Comment by paetling** [comment]
seems to me that you would want to also make the system use the new always return payload.  Are you just waiting to do that in a new pr?

**Comment by jsingle** [comment]
I wasn't going to but maybe that would make sense.  I was thinking that for commands that run entirely on the client, they can just run inline without doing a payload

File: dusty/payload.py [hunk 1]
```diff
@@ -11,6 +12,9 @@ def __eq__(self, other):
             return self.fn == other.fn and self.args == other.args and self.kwargs == other.kwargs
         return False
 
+    def run(self):
+        self.fn(self.args, self.kwargs)
```
Comment (thieman) [inline dusty/payload.py]: Don't these need various *'s

**Comment by paetling** [comment]
I like the idea of standardizing on one way of handling commands (always returning a payload) and then the client can handle the rest

File: dusty/payload.py (hunk 1 above)
Comment (jsingle) [inline dusty/payload.py]: yep good catch

Comment (paetling) [inline dusty/payload.py]: He is right. *args, **kwargs

**Comment by jsingle** [comment]
OK I'll stick that in this PR then

**Comment by paetling** [comment]
:ghost: :cow:

File: dusty/cli/__init__.py [hunk 2]
```diff
@@ -102,6 +102,20 @@ def _connect_to_daemon():
     sock.settimeout(None)
     return sock
 
+def _run_payload(payload):
+    if payload.run_on_daemon:
+        sock = _connect_to_daemon()
+        if sock is None:
+            return 1
```
Comment (thieman) [inline dusty/cli/__init__.py]: This is weird.

**Pull request merged**